src/flat-button.jsx

```diff
@@ -81,7 +84,7 @@ const FlatButton = React.createClass({
   getDefaultProps() {
     return {
       labelStyle: {},
-      labelPosition: 'before',
+      labelPosition: 'after',
```

Comment:
Ooh, could we do something to make it backwards compatible? 

@oliviertassinari I feel like I've seen quite a few PRs lately with breaking changes. Feel free to create a new branch (`v0.14-temp` or something like that) off of master to merge PRs with breaking changes. However, we should still aim for making those changes non-breaking if we can..